manim/mobject/mobject.py

```diff
@@ -2128,12 +2128,75 @@ def set_z_index_by_z_coordinate(self):
 
 
 class Group(Mobject):
-    """Groups together multiple Mobjects."""
+    """A group of mobjects.
+
+    This can be used to group multiple :class:`~.Mobject` instances together
+    in order to scale, move, or perform other animations on them together.
+
+    To add :class:`~.Mobject`s to a :class:`~.Group`, you can either use the
+    :meth:`~.Group.add` method, or use the `+` and `+=` operators. Similarly, you
+    can subtract elements of a Group via :meth:`~.Group.remove` method, or
+    `-` and `-=` operators:
+
+    See Also
+    --------
+    :class:`VGroup`
+    """
 
     def __init__(self, *mobjects, **kwargs):
         Mobject.__init__(self, **kwargs)
         self.add(*mobjects)
 
+    def __repr__(self):
+        return (
+            self.__class__.__name__
+            + "("
+            + ", ".join(str(mob) for mob in self.submobjects)
+            + ")"
+        )
+
+    def __str__(self):
+        return (
+            f"{self.__class__.__name__} of {len(self.submobjects)} "
+            f"submobject{'s' if len(self.submobjects) > 0 else ''}"
+        )
+
+    def add(self, *mobjects):
+        """Checks if all passed elements are an instance of Mobject and then add them to submobjects
+
+        Parameters
+        ----------
+        mobjects : :class:`~.Mobject`
+            List of Mobjects to add
+
+        Returns
+        -------
+        :class:`Group`
```

Comment:
One of my changes from VGroup that does have the `~.` -- should be fixed in VGroup as well.